src/plots/cartesian/dragbox.js

```diff
@@ -703,13 +703,14 @@ function makeDragBox(gd, plotinfo, x, y, w, h, ns, ew) {
             for(i = 0; i < axList.length; i++) {
                 ax = axList[i];
 
-                if(!ax._rangeInitial) {
-                    attrs[ax._name + '.autorange'] = true;
-                }
-                else {
-                    rangeInitial = ax._rangeInitial;
-                    attrs[ax._name + '.range[0]'] = rangeInitial[0];
-                    attrs[ax._name + '.range[1]'] = rangeInitial[1];
+                if(!ax.fixedrange) {
```

Comment:
I thought we may need to check for having zero in the first element of range array here.
Here is a [demo](https://codepen.io/MojtabaSamimi/pen/vvyqgJ?editors=0010) which fails using with `yaxis2.range=[0,1]`.

Reply:
The demo is actually working with [0,1]. So need to worry there.